tableauserverclient/server/endpoint/auth_endpoint.py

```diff
@@ -73,3 +73,12 @@ def switch_site(self, site_item):
         self.parent_srv._set_auth(site_id, user_id, auth_token)
         logger.info('Signed into {0} as user with id {1}'.format(self.parent_srv.server_address, user_id))
         return Auth.contextmgr(self.sign_out)
+
+    @api(version="3.10")
+    def revoke_all_server_admin_tokens(self):
+        url = "{0}/{1}".format(self.baseurl, 'revokeAllServerAdminTokens')
+        if not self.parent_srv.is_signed_in():
```

Comment:
This check is necessary, since it'll just be a silent operation if a user isn't signed in. The post_request call below should throw an error telling the user to first sign in.

Reply:
sorry, I meant to say that the check is not necessary 😓 it's redundant since post_request() will check anyways

Reply:
I just removed the check from the most recent commit. Thanks for addressing that!